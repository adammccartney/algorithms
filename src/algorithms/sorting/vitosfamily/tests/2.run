./vito < tests/2.in
