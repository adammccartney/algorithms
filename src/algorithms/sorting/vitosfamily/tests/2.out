2
4
16, 50, 11936, 12322, 13529, 13663, 15412, 15687, 15975, 16057, 16627, 17065, 18895, 19535, 19799, 19858, 24059
